cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -810,7 +811,17 @@ subroutine evp (dt)
                enddo  ! iblk
                !$OMP END PARALLEL DO
 
-            elseif (grid_ice == "C") then
+               ! U fields at NE corner
+               ! calls ice_haloUpdate, controls bundles and masks
+               call dyn_HaloUpdate (halo_info,          halo_info_mask,    &
```

Comment:
Not something new in this PR, but I'm noticing that all calls to `dyn_haloUpdate` in `ice_dyn_evp::evp` use this capitalization: `dyn_HaloUpdate` (with a capital `H`), whereas the subroutine is defined in `ice_dyn_shared` as `dyn_haloUpdate` (small `h`). Not that big of a deal but it could be nice to fix that .

Reply:
which is preferred?

Reply:
I'm not sure if we have a firm standard or not. but for now I think we should use the case that is used when defining the subroutine (`dyn_haloUpdate`).

Reply:
I'm not too worried about this, but happy to go the way @phil-blain suggests.

Reply:
I agree with @phil-blain, it's better to be consistent to make grepping just a little easier.  We aren't completely consistent now but can work toward it.

Reply:
`grep` and `git grep` were exactly what I had in mind when mentioning it (in fact that's how I noticed!)